MovieList/MovieList/MovieList/MovieListCoordinator.swift

```diff
@@ -32,7 +32,7 @@ extension MovieListCoordinator: MovieListVCDelegate {
                     self.movieController.updateUI()
 
                 case .failure(_):
-                    // TODO: pass alert back to MovieListVC.
+                self.movieController.networkCallAlert()
                     break
```

Comment:
You don't need the `break` in the switch and I _think_ indentation is wrong. Ctrl + I on the whole switch should fix it.

Reply:
Removed the 'break' and re-indented the file.